### ISSUE TITLE
Random Arcana

### ISSUE THREAD

**Issue by FedericoBohle** [ValveSoftware/Dota2-Gameplay#4209]
Cant choose my arcana and instead got a random one

**Comment by jackgaracochea** [comment]
This happened to me as well with the same skin, I'm not really sure how and was surprised when other's said they got to pick their arcana

**Comment by FedericoBohle** [comment]
exactly! i was happy cause i got a free arcane, then check whatsapp group i found out that the boys get to pick theirs
